notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -116,39 +116,6 @@
         "to generate a cost estimate based on your projected usage."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    models = aiplatform.Model.list(filter="display_name=gsod_unique")
</pre><p>This won't work. Save the display_name to a variable and pass that in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>